### PR TITLE
ceph: Fix how ceph CLI args are parsed

### PR DESCRIPTION
**Pull request by mvollman**
The `strContainsClient` function is splitting characters instead of words and not correctly determining if a user has been defined.

I mistakenly closed the original merge request #1083 while rebasing.

### CODE REVIEW

**Comment by akutz** [comment]
Hi @mvollman,

Unless the commits need to remain separate, do you mind please squashing them? Thanks!

File: libstorage/drivers/storage/rbd/utils/utils_test.go [hunk 1]
```diff
@@ -75,3 +75,43 @@ func TestParseMonitorAddresses(t *testing.T) {
...
+			args:    "--user myuser",
+			present: true,
+		},
+		{
+			args:    "--name client.myuser",
+			present: true,
+		},
+		{
+			args:    "--cluster cluster2",
+			present: true,
```
Comment (codenrhoden) [inline libstorage/drivers/storage/rbd/utils/utils_test.go]: This should be `present: false`. There is only a `cluster` flag in here, and therefore no client is detected. This should actually fail if you run the tests, e.g. :
`go test -v github.com/thecodeteam/rexray/libstorage/drivers/storage/rbd/utils`

Do you not see the tests fail?

I also had a 6th variation in the sample I pasted:
```
		{
			args:    "--cluster cluster2 --id ceph",
			present: true,
		},
```

That was just to make sure we catch a mixed example.

**Comment by mvollman** [comment]
@codenrhoden What is the one outstanding request?

**Comment by codenrhoden** [comment]
@mvollman Nothing! You took care of it in your most recent commit.  Thank you!

**Pull request merged**